.github/workflows/run-eslint.yaml

```diff
@@ -12,5 +12,17 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v4
 
+    - name: Get changed files
+      id: changed-files-specific
+      uses: tj-actions/changed-files@90a06d6ba9543371ab4df8eeca0be07ca6054959
+      with:
+        files: |
+          .github/**
+          testing/e2e/skr/**
+          scripts/**
+          Makefile
+          **/*.sh
```

Comment:
I added this to make sure the jobs run in case we add a script to this workflow in the future, but for now it is indeed unnecessary and I can remove it.